src/components/Button.js

```diff
@@ -245,7 +250,8 @@ class Button extends React.Component {
       },
       buttonText: {
         marginLeft: (this.props.isActive && this.props.actionText) ? 10 : 0
-      }
+      },
+      ...this.props.styles
```

Comment:
Should the the styles prop have a default value of `{}`? Or does spread do fine if a value is null/undefined?

Reply:
I think it works okay:
![screen shot 2018-07-11 at 1 34 28 pm](https://user-images.githubusercontent.com/2127504/42595323-2cdb53bc-850f-11e8-8010-7758ad4ee815.png)

Reply:
This now has a default of `{}`